frontend/app/components/work-packages/wp-single-view/wp-single-view.directive.html

```diff
@@ -102,6 +102,6 @@ <h3 class="attributes-group--header-text" ng-bind="$ctrl.text.idLabel"></h3>
 
   <wp-attachments
       work-package="$ctrl.workPackage"
-      hide-empty-fields="$ctrl.hideEmptyFields">
+      data-ng-show="$ctrl.filesExist()">
```

Comment:
Wouldn't this cause the attachment section to never show initially despite `show all` selected?